src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -215,8 +215,8 @@ public String getRepoOwner() {
     }
 
     @NonNull
-    public List<SCMTrait<?>> getTraits() {
-        return Collections.<SCMTrait<?>>unmodifiableList(traits);
+    public List<SCMTrait<? extends SCMTrait<?>>> getTraits() {
+        return Collections.<SCMTrait<? extends SCMTrait<?>>>unmodifiableList(traits);
```

Comment:
Are these changes simply mandated by an `scm-api` update to a released version, before proceeding with your changes?